src/components/HomePage/ContactSection.spec.js

```diff
@@ -0,0 +1,59 @@
+import React from "react";
+import ContactSection from "./ContactSection";
+import ReactDOM from "react-dom";
+import { render, cleanup } from "@testing-library/react";
+// import HomePage from "../../pages/HomePage";
+
+afterEach(cleanup);
+
+describe("ContactSection", () => {
+    describe("render", () => {
+        it("renders without crashing", () => {
+            const div = document.createElement("div");
+            ReactDOM.render(<ContactSection />, div);
+        });
```

Comment:
You don't need this test, if it somehow fails, the other tests will fail as well